src/components/screens/multiSignature/result/result.js

```diff
@@ -36,10 +38,11 @@ const Result = ({
   }, [transactionInfo]);
 
   return (
-    <section className="transaction-status">
+    <section className={`${styles.wrapper} transaction-status`}>
       <div className={styles.header}>
         <h1>{t('Register multisignature account')}</h1>
       </div>
+      <ProgressBar current={4} />
```

Comment:
Should not be 3?

Reply:
Technically yes but in the design, when we are at the 3rd step, we want to style as if it were an older step (like step would be styled when we are at step 2). Thats why, I used 4 here.